docs/serving/autoscaling/README.md

```diff
@@ -11,7 +11,9 @@ For more information about which metrics can be used to control the Autoscaler,
 
 ## Optional autoscaling configuration tasks
 
-* Configure your Knative deployment to use the Kubernetes [Horizontal Pod Autoscaler (HPA)](../../install/any-kubernetes-cluster.md#optional-serving-extensions) instead of the default KPA.
+* Configure your Knative deployment to use the Kubernetes Horizontal Pod Autoscaler (HPA)
+instead of the default KPA.
+For how to install HPA, see [Install optional Eventing extensions](../../install/install-extensions.md#install-optional-serving-extensions).
```

Comment:
It's odd to me to add an additional line for this rather than how it was formatted before, I think it's less readable.
It might be worth checking out "minimalism in tech writing" as a concept if you haven't come across this before, a quick google search will give a lot of links on the topic.

Reply:
I've used the standard cross referencing format from the Google style guide. I am aware of minimalism, but, when following a link, readers should have a reasonable expectation of what they will find at the target.

Readers need to be aware that, after following this link, they will find info about _installing_ HPA and not general information about HPA.

Reply:
I disagree, but not strongly enough to block a PR on it 🙂